ynr/apps/candidates/management/commands/candidates_create_csv.py

```diff
@@ -73,6 +74,16 @@ def handle(self, **options):
                 membership_by_election.get(election.slug, []),
             )
 
+        # Make a CSV file per election date
+        slugs_by_date = defaultdict(list)
+        for slug in membership_by_election.keys():
+            slugs_by_date[slug.split(".")[-1]].append(slug)
```

Comment:
I know the end of the slug _should_ always be the date, but any particular reason for parsing the date out of the slug here instead of using `Election.election_date` ?

Reply:
Because `membership_by_election` is just a dict of other lists/dicts without the `Election` model anywhere. I could do another query to aggregate all elections by date and slug and pull the slugs out of those IDs, but I figured this way was easier. 

What do you reckon?

Reply:
meh - should be fine. Just checking..